src/frontend/planner_test/tests/testdata/subquery_expr_correlated.yaml

```diff
@@ -5,19 +5,19 @@
     select * from t1 where x > (select 1.5 * min(x) from t2 where t1.y=t2.y and t2.y = 1000)
   logical_plan: |
     LogicalProject { exprs: [t1.x, t1.y] }
-    └─LogicalFilter { predicate: (t1.x > (1.5:Decimal * min(t2.x))) }
+    └─LogicalFilter { predicate: (t1.x > $expr1) }
       └─LogicalApply { type: LeftOuter, on: true, correlated_id: 1, max_one_row: true }
         ├─LogicalScan { table: t1, columns: [t1.x, t1.y, t1._row_id] }
-        └─LogicalProject { exprs: [(1.5:Decimal * min(t2.x))] }
+        └─LogicalProject { exprs: [(1.5:Decimal * min(t2.x)) as $expr1] }
           └─LogicalAgg { aggs: [min(t2.x)] }
             └─LogicalProject { exprs: [t2.x] }
               └─LogicalFilter { predicate: (CorrelatedInputRef { index: 1, correlated_id: 1 } = t2.y) AND (t2.y = 1000:Int32) }
                 └─LogicalScan { table: t2, columns: [t2.x, t2.y, t2._row_id] }
   optimized_logical_plan: |
-    LogicalJoin { type: Inner, on: IsNotDistinctFrom(t1.y, t1.y) AND (t1.x::Decimal > (1.5:Decimal * min(t2.x))), output: [t1.x, t1.y] }
-    ├─LogicalProject { exprs: [t1.x, t1.y, t1.x::Decimal] }
+    LogicalJoin { type: Inner, on: IsNotDistinctFrom(t1.y, t1.y) AND ($expr31 > $expr32), output: [t1.x, t1.y] }
```

Comment:
The numbers seem to start from 31. Perhaps need to reset `next_expr_display_id` every time?

Reply:
Currently, the expression id is generated and stored in the project schema's field name, so that upper plan node can print the correct name. It seems reset doesn't help.